pyomo/contrib/parmest/parmest.py

```diff
@@ -453,13 +914,7 @@ def _instance_creation_callback(self, experiment_number=None, cb_data=None):
         return model
 
     def _Q_opt(
-        self,
-        ThetaVals=None,
-        solver="ef_ipopt",
-        return_values=[],
-        bootlist=None,
-        calc_cov=False,
-        cov_n=None,
+        self, ThetaVals=None, solver="ef_ipopt", return_values=[], bootlist=None
```

Comment:
Did you run black? I suspect your change is undoing some black formatting edits.